common/src/main/scala/com/gu/media/upload/actions/UploadActionHandler.scala

```diff
@@ -102,21 +102,24 @@ abstract class UploadActionHandler(store: UploadsDataStore, plutoStore: PlutoDat
   }
 
   private def sendToPluto(upload: Upload): Unit = {
-
     val plutoData: PlutoSyncMetadata = upload.metadata.pluto
 
-    plutoData.projectId match {
-      case _ if plutoData.assetVersion == -1 =>
+    if(plutoData.enabled) {
+      log.info(s"Not syncing to Pluto upload=${upload.id} atom=${plutoData.atomId}")
+    } else {
+      plutoData.projectId match {
+        case _ if plutoData.assetVersion == -1 =>
         // TODO: work out what to do here? probably need to manually add the asset
```

Comment:
We do still need it but the pluto stuff doesn't need to care. I've removed that case